pkg/operator/controller.go

```diff
@@ -491,3 +499,33 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 
 	return nil
 }
+
+func (c *Controller) hasTrustedCA() bool {
+	_, err := os.Stat("/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem")
+	return err != nil
```

Comment:
if err == nil then we have the trusted ca, no? it seems like you are doing the opposite here.